source/fundamentals/linq.txt

```diff
@@ -497,6 +497,82 @@ The following is a subset of the returned results:
        ]
     }
 
+$vectorSearch
+~~~~~~~~~~~~~
+
+The ``$vectorSearch`` aggregation stage performs an *approximate nearest neighbor* search
+on an Atlas Vector Search index on the specified field.
```

Comment:
I think "peforms ... search on a vector in the specified field." would be more accurate?
Like [here](https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-stage/#mongodb-pipeline-pipe.-vectorSearch).
Also worth specify the need for vector index explicitly (besides Tip)?